src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -63,17 +60,95 @@ import play.api.libs.json._
   * Marathon will notice spurious tasks in the offer and create the appropriate
   * unreserve operations. See https://github.com/mesosphere/marathon/issues/3223
   */
```

Comment:
maybe the docs here are obsolete now and should be updated?

Reply:
Already done, but not committed and pushed yet :)